src/config/ConfigManager.cpp

```diff
@@ -208,7 +208,8 @@ void CConfigManager::setDefaultVars() {
     configValues["master:smart_resizing"].intValue         = 1;
     configValues["master:drop_at_cursor"].intValue         = 1;
 
-    configValues["animations:enabled"].intValue = 1;
+    configValues["animations:enabled"].intValue       = 1;
+    configValues["animations:workspace_gap"].intValue = 0;
```

Comment:
does this really belong to animations: ?

Reply:
Yes, because the gap is only shown when switching workspaces. I only modified workspace change anims.
An animation option, technically at least. 
Do you think we should have this in general or something?

Reply:
moved to general:gaps_workspace in the latest commit